main.py

```diff
@@ -69,8 +73,13 @@ def export_streams_to_clipboard(self):
         clipboard.clear(mode=clipboard.Clipboard)
         clipboard.setText(text, mode=clipboard.Clipboard)
 
-    def add_new_stream(self, *args, stream_url=None, stream_quality="best"):
+    def add_new_stream(self, *args, stream_url=None, stream_quality=cfg[CONFIG_QUALITY]):
         """Adds a new player for the specified stream in the grid."""
+        stream_url, status = QtWidgets.QInputDialog.getText(self, "Stream input", "Enter the stream URL:")
+        if not status:
+            return
+
+        # TODO: Add streams here.
```

Comment:
Could it be that you merged against your local `master` branch which wasn't updated and not `origin/master`? @kaszim